### DISCUSSION TITLE
Invalid key for Google drive application

### DISCUSSION THREAD

**Discussion by FelipeCODX** [JoePenna/Dreambooth-Stable-Diffusion#84] [General]
I'm getting the following error when trying to upload my trained model to google drive, something related to an invalid OAuth method [(error URL)](https://mail.google.com/mail/u/0/?tab=rm&ogbl#inbox)
P.S.: Olá, Sr. Felipe aleatório aqui, kkkkk.

![image](https://user-images.githubusercontent.com/34315431/197106593-6b031f67-2556-4c62-8f32-055d5a6b6da9.png)

**Comment by ylucic** [comment]
Same Issue. I'm using vast.ai

**Comment by FelipeCODX** [comment]
[SOLVED]
I created a secondary brand new google drive account and it worked fine with it, not sure if it gonna be replicable tho

**Reply by ylucic** [reply]
making a new account worked.

**Reply by chakalakasp** [reply]
For what it's worth that error is because the auth method this is trying to use is deprecated and switched off -- surprised a new account works
